projects/perception/object_detection_2d/centernet/inference_tutorial.ipynb

```diff
@@ -0,0 +1,263 @@
+   "metadata": {},
+   "source": [
+    "# CenterNet Tutorial\n",
+    "\n",
+    "This notebook provides a tutorial for running inference on a static image in order to detect objects.\n",
+    "The implementation of the [CenterNetDetectorLearner](https://github.com/opendr-eu/opendr/blob/master/docs/reference/object-detection-2d-centernet.md) is largely copied from its original\n",
+    "[implementation](https://github.com/dmlc/gluon-cv/tree/master/scripts/detection/center_net).\n",
+    "More information on modifications and license can be found\n",
+    "[here](https://github.com/opendr-eu/opendr/blob/master/src/opendr/perception/object_detection_2d/centernet/README.md)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we need to import the learner and initialize it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/administrator/opendr/venv/lib/python3.8/site-packages/gluoncv/__init__.py:40: UserWarning: Both `mxnet==1.8.0` and `torch==1.7.1` are installed. You might encounter increased GPU memory footprint if both framework are used at the same time.\n",
+      "  warnings.warn(f'Both `mxnet=={mx.__version__}` and `torch=={torch.__version__}` are installed. '\n"
+     ]
+    }
+   ],
+   "source": [
+    "from opendr.perception.object_detection_2d.centernet.centernet_learner import CenterNetDetectorLearner\n",
+    "\n",
+    "learner = CenterNetDetectorLearner(device='cuda', backbone='resnet50_v1b')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that we can alter the device (e.g., 'cpu', 'cuda', etc.)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After creating our model, we need to download pre-trained weights."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "learner.download(\".\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "and load our weights:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "[11:22:20] ../src/base.cc:80: cuDNN lib mismatch: linked-against version 8101 != compiled-against version 8004.  Set MXNET_CUDNN_LIB_CHECKING=0 to quiet this warning.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Loaded mxnet model.\n"
+     ]
+    }
+   ],
+   "source": [
+    "learner.load(\"./centernet_default\", verbose=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will also download one sample image and format it so we can use it in opendr for testing:"
```

Comment:
```suggestion
    "We will also download one sample image and format it so we can use it in OpenDR for testing:"
```

Reply:
Fixed with [d97cd1b](https://github.com/opendr-eu/opendr/pull/230/commits/d97cd1b42f9fdd581b36cb3bddc07e94dde38fdf), for some reason I could not automatically commit suggestions